communicate-on-pull-request-released/__tests__/main.test.ts

```diff
@@ -0,0 +1,90 @@
+const path = require('path');
+const nock = require('nock');
+
+const validScenarios = [
+  {
+    response: 'release.json',
+    event_name: 'release'
+  }
+];
+
+const invalidScenarios = [
+  {
+    response: 'pull-request-closed.json'
+  },
+  {
+    response: 'action-created.json'
+  },
+  {
+    response: 'release-missing-pr-numbers.json',
+    event_name: 'release'
+  }
+];
+
+describe('action test suite', () => {
+  for (const scenario of validScenarios) {
+    it(`It posts a comment on pull requests, referenced issues and update labels for (${scenario.response})`, async () => {
+      process.env['INPUT_REPO-TOKEN'] = 'token';
+      process.env['INPUT_PR-LABEL-TO-ADD'] = 'label-to-add';
+      process.env['INPUT_PR-LABEL-TO-REMOVE'] = 'label-to-remove';
+
+      process.env['GITHUB_REPOSITORY'] = 'foo/bar';
+      process.env['GITHUB_EVENT_NAME'] = scenario.event_name;
+      process.env['GITHUB_EVENT_PATH'] = path.join(
+        __dirname,
+        scenario.response
+      );
+
+      const api = nock('https://api.github.com')
+        .persist()
+        .post(
+          '/repos/foo/bar/pulls/999/reviews',
+          '{"body":"Congratulations! :tada: This was released as part of [_fastlane_ 2.134.1](https://github.com/Codertocat/Hello-World/runs/128620228) :rocket:","event":"COMMENT"}'
```

Comment:
Are we currently using `_fastlane_` (vs. `fastlane`)?

Reply:
Yes, we are 

https://github.com/fastlane/issue-bot/blob/457348717d99e5ffde34ca1619e7253ed51ec172/bot.rb#L271